forum/qa-plugin/tips-widget/tips-widget-admin.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+class tips_widget_admin
+{
+	public function option_default($option)
+	{
+		$response = null;
+		
+		if ($option === 'tips-enable')
+		{
+			$response = false;
+		}
+		elseif ($option === 'tips-widget-content' || $option === 'tips-page-content')
+		{
+			$response = file_get_contents(__DIR__ . '/snippets/' . $option . '.html');
+			if ($response === false)
+			{
+				$response = '';
+			}
+		}
+		
+		return $response;
+	}
+	
+	public function admin_form(&$qa_content)
+	{	
+		$saved = qa_clicked('tips-save');
+		
+		if ($saved)
```

Comment:
true === $saved

Reply:
Ta zmienna zawsze posiada wartości 0 lub 1 (albo false lub true, bo nie jestem pewien, jak to Q2A trzyma), więc w tym przypadku taki zapis jest zbędny, ale rozumiem, że w niczym by nie przeszkodził.

Reply:
To nie jest związane z samym kodem tylko dobrą praktyką. PHP nie ma silnego typowania i takie zapisy zabezpieczają nas przed błędami które są trudne do wykrycia. Tylko o to chodzi :)

Reply:
Dlatego wolałbym żeby pojawił się ten zapis - wtedy dam pełen aprove dla review. Mimo, że nie podoba mi się formatowanie

Reply:
Całkowicie rozumiem ten zapis i nie neguję go, lecz w tym przypadku, gdy jestem pewien wartości, mogę napisać prosty warunek. Po prostu obaj mamy w tym przypadku rację. Ty wolisz pisać zawsze bezpieczny kod, nawet jeśli w danym przypadku nie jest to obligatoryjne, ja natomiast używam zabezpieczeń tylko tam, gdzie są konieczne.

Reply:
No ale ta "pewność" czasami może zawieść, więc "twoja racja jest bardziej twojsza", ale minimalnie xD

Reply:
> Ta zmienna zawsze posiada wartości 0 lub 1 (albo false lub true, bo nie jestem pewien, jak to Q2A trzyma), więc w tym przypadku taki zapis jest zbędny, ale rozumiem, że w niczym by nie przeszkodził.

Chyba jednam nie mam racji. Zapis z === zawsze by zwracał false dla 0 i 1 lub '0' i '1'. Jako że nie jestem pewien typu, pozostała taka, a nie inna forma.

Reply:
Dobra, ta funkcja zwraca true lub false. No to mogę poprawić, jeśli ma to zaważyć nad tym PR.